apps/server/src/core/logger/interfaces/logger-config.ts

```diff
@@ -2,4 +2,5 @@ import { AvailableLogLevel } from './logger.interface';
 
 export interface ILoggerConfig {
 	AVAILABLE_LOG_LEVELS: AvailableLogLevel[];
+	LOG_LEVEL: string;
```

Comment:
rename to NEST_LOG_LEVEL

Reply:
should Nest have its own log level independent from the global log level?

Reply:
after discussing with @SevenWaysDP, we decided to keep the NEST_LOG_LEVEL for now. Otherwise we run into issues with the logger in feathers.